source/Engine.cpp

```diff
@@ -1065,7 +1065,7 @@ void Engine::Draw() const
 		Point size(highlightSprite->Width(), highlightSprite->Height());
 		const Color &color = *colors.Get("flagship highlight");
 		// The flagship is always in the dead center of the screen.
```

Comment:
Well this comment isn't quite true anymore.

Reply:
```suggestion
```